### DISCUSSION TITLE
Whisper Splits Phrases

### DISCUSSION THREAD

**Discussion by p1ke73** [openai/whisper#1260] [Q&A] [answered]
If audio file is long (more than 20min), suddenly whisper makes too many timestamps from approximately 20min and phrasess are divided into words like this:

[00:19:34] I met a very nice friend in a college.
[00:20:2] His name
[00:20:3] is
[00:20:5] John

From the beginning, whisper doesn't splits phrases, but from some moment, starts doing so. I checked if the word level timestamp option is on, but it was turned off. Are there any ways to fix this?

**Comment by jongwook** [comment, marked as answer]
There's a certain level of randomness in Whisper's behavior in splitting segments, and unfortunately it's path dependent so that when it starts to happen, it tends to continue happening. In the case of excessively splitting phrases like the above, an easy fix is to post-process the transcript by merging lines that are too short ..
